pygorithm/fibonacci/space_optimized.py

```diff
@@ -0,0 +1,44 @@
+# Author: Ashutosh Gupta
+# Created On: 8/15/2017
+# Time: 3:24 PM
+
+"""
+nth fibonacci number - Space Optimized
```

Comment:
Can we call it `Fibonacci implementation through using looping technique (space optimized)` or something else?